routes/products.html

```diff
@@ -0,0 +1,510 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://subtleui.netlify.app/styles/styles.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/12af55b294.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../styles/styles.css" />
+    <link rel="stylesheet" href="../styles/products.css" />
+    <title>Sneakoholics | Products</title>
+  </head>
+  <body>
+    <nav class="navbar fixed">
+      <a href="#" class="nav-heading">Sneakoholics</a>
+      <input
+        type="text"
+        class="input"
+        placeholder="What do you want to wear today?"
+      />
+      <ul class="nav-actions">
+        <li class="nav-action-item">
+          <button class="btn btn-primary">
+            <a href="./login.html">Login</a>
+          </button>
+        </li>
+        <li class="nav-action-item">
+          <div class="badge-container">
+            <i class="fas fa-shopping-cart fa-2x" aria-hidden="true"></i>
+            <span class="badge badge-icon">10</span>
+          </div>
+        </li>
+        <li class="nav-action-item">
+          <div class="badge-container">
+            <i class="far fa-heart fa-2x"></i>
+            <span class="badge badge-icon">10</span>
+          </div>
+        </li>
+      </ul>
+    </nav>
+    <main>
+      <aside class="filters">
+        <ul class="filter-list">
+          <li class="filter-item">
+            <div class="flex justify-between align-ctr">
+              <div class="filter-heading">Filters</div>
+              <button class="btn btn-link">Clear</button>
+            </div>
+          </li>
+          <li class="filter-item">
+            <div class="filter-heading">Price</div>
+            <input id="slider" type="range" min="100" max="50000" />
```

Comment:
you can also use `step` attribute, along with `<datalist>` element to add breakpoints in your slider.
Refer to MDN docs